### PR TITLE
More-Info Story Branch

### PR DESCRIPTION
**Pull request by avasilescu**
The previous pull request got cluttered with 1000 commits because I had to rebase the branch to correct my email address. Obviously, that did not work out so well as it wanted to submit 1000+ commits. I probably did something wrong. This should be a lot cleaner as there are only 4 commits to be committed. 1 commit for each file (separate story) being added.

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
Thanks, @avasilescu, and sorry you had trouble with the rebase. @casey-collab, please take a look.

File: english/light-fire/run-away/shout/more-info/more-info.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+You ask him "What is going on? Where am I?"
```
Comment (casey-collab) [inline english/light-fire/run-away/shout/more-info/more-info.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by avasilescu** [comment]
Hah. I guess the link to the original page I created got lost during the rebase/delete and start-over portion. Also shout out from another CMUer in the Bay Area.

**Comment by cbuckey-uda** [comment]
Oh nice! @casey-collab, please take another look.

**Comment by casey-collab** [comment]
Looks great! :ferris_wheel:

**Pull request merged**